// Copyright 2018 Contributors to Hyperledger Sawtooth
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
// -----------------------------------------------------------------------------

syntax = "proto3";


message UserContainer {

   repeated User users = 2;

}


message User {

    string user_id = 1;

    string name = 2;

    string manager_id = 3;

    string metadata = 4;

    string email = 5;

    string remote_id = 6;

    string username = 7;

    int64 created_date = 8;

    string distinguished_name = 9;

}
